src/DssSpell.sol

```diff
@@ -96,10 +96,10 @@ contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
         address CLIPPER_MOM = DssExecLib.clipperMom();
         address MCD_END = DssExecLib.end();
 
-        // ---- Revoke OsmMom to access to the Old PIP
+        // Revoke OsmMom to access the Old OSM
```

Comment:
Very minor thing, but we call it here `OSM` and then `CRVV1ETHSTETH-A PIP`. I'd used the same term for both comments

Reply:
done in c0a7c68.